src/yarp.c

```diff
@@ -6352,6 +6352,11 @@ parse_expression_prefix(yp_parser_t *parser, binding_power_t binding_power) {
       yp_token_t opening = parser->previous;
       yp_node_t *array = yp_array_node_create(parser, &opening, &opening);
 
+      // skip all leading whitespaces
+      while (parser->current.type == YP_TOKEN_WORDS_SEP) {
```

Comment:
I think there should only be a single token here, so you should be able to do just `accept(parser, YP_TOKEN_WORDS_SEP);` without the look